flink-streaming-java/src/main/java/org/apache/flink/streaming/api/CheckpointingMode.java

```diff
@@ -90,4 +90,16 @@ public static org.apache.flink.core.execution.CheckpointingMode convertToCheckpo
                 throw new IllegalArgumentException("Unsupported semantic: " + semantic);
         }
     }
+
+    public static org.apache.flink.streaming.api.CheckpointingMode convertFromCheckpointingMode(
```

Comment:
I guess it should also be deprecated

Reply:
Ah, yes, but the whole class`org.apache.flink.streaming.api.CheckpointingMode` is already annotated with `@Deprecated` so I guess there is no need to annotate this method separately?